test/common/wpt.js

```diff
@@ -458,8 +458,16 @@ class StatusLoader {
 
   load() {
     const dir = path.join(__dirname, '..', 'wpt');
-    const statusFile = path.join(dir, 'status', `${this.path}.json`);
-    const result = JSON.parse(fs.readFileSync(statusFile, 'utf8'));
+    let statusFile = path.join(dir, 'status', `${this.path}.json`);
+    let result;
+
+    if (fs.existsSync(statusFile)) {
+      result = JSON.parse(fs.readFileSync(statusFile, 'utf8'));
```

Comment:
Can't JSON also be `require`d?

Reply:
Yeah. But this isn't changed from the way it was and I'm not about to make an unnecessary change and go through another round of ci retries and resumes.